microsoft-edge/accessibility/design.md

```diff
@@ -11,32 +11,44 @@ keywords: accessibility, accessibility for developers, accessible websites, edge
 ---
 # Designing accessible websites
 
-Creating an inclusive design makes technology usable by all people no matter their age, education, geographic location, language, or disability.  People using technology and browsing the web have a wide range of abilities and preferences.  As you design your website, keep in mind the following key accessibility scenarios:
+Creating an inclusive design makes technology usable by all people no matter their age, education, geographic location, language, or disability.  People using technology and browsing the web have a wide range of abilities and preferences.  As you design your website, keep in mind the following key accessibility scenarios.
 
-*   Screen readers—Users who are blind or visually impaired rely on screen readers to interpret and interact with the UI of your app.  Interpreting involves reading the UI element names, roles, values, and so on, and interacting with the UI involves moving the focus from one element to another and invoking functionality.
-*   Keyboard accessibility—Many accessibility users rely on the keyboard to navigate and operate the UI by:
-    *   Moving focus among elements by using the Tab key.
-    *   Navigating in container elements such as lists, grids, and tree views by using the arrow keys.
-    *   Activating functionality \(invoking actions\) by using the Enter or Space key.
-    *   Using shortcut keys to efficiently access other app functionality.
-*   Accessible visual experience—Users who are visually impaired need a sufficient text contrast ratio for text content, and a good visual experience with high contrast themes overall.  Users who are color blind need information to be conveyed in ways other than through color.
 
-Many common accessibility issues on the web can be solved through good coding practice.  The [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20) documentation provides techniques and best practices to help you design more accessible dynamic web applications.  For more information, navigate to [Building accessible websites](./build/index.md).
+<!-- ====================================================================== -->
+## Screen readers
 
-## Resources
+Users who are blind or visually impaired rely on screen readers to interpret and interact with the UI of your app.  Interpreting involves reading the UI element names, roles, values, and so on, and interacting with the UI involves moving the focus from one element to another and invoking functionality.
```

Comment:
```suggestion
Users who are blind or visually impaired rely on screen readers to interpret and interact with the UI of your app.  Interpreting involves reading the UI element names, roles, values, and so on. Interacting with the UI involves moving the focus from one element to another and invoking a function.
```

Reply:
@vchapel - ended up with this.  Broke up sentence per Comment:

Users who are blind or visually impaired rely on screen readers to interpret and interact with the UI of your app.  Interpreting involves reading the UI element names, roles, and values.  Interacting with the UI involves moving the focus from one element to another and invoking functionality such as clicking an element.